src/component/registry/bpmn-elements-registry.ts

```diff
@@ -112,6 +112,22 @@ export class BpmnElementsRegistry {
       this.bpmnModelRegistry.refreshCell(bpmnElementId, this.cssRegistry);
     });
   }
+
+  /**
+   * Remove one/several CSS class(es) from one/several BPMN element(s).
+   *
+   * @param bpmnElementIds The BPMN id of the element(s) where to remove the CSS classes
+   * @param classNames The name of the class(es) to remove from the BPMN element(s)
+   */
+  removeCssClasses(bpmnElementIds: string | string[], classNames: string | string[]): void {
+    // TODO duplication with addCssClasses
+    // only the call to remove or add class differ
+    ensureIsArray<string>(bpmnElementIds).forEach(bpmnElementId => {
+      if (this.cssRegistry.removeClassNames(bpmnElementId, ensureIsArray<string>(classNames))) {
```

Comment:
To avoid to call **ensureIsArray** method for each **bpmnElementId**, I think it's better to call it one time before first **ensureIsArray** calling like done in **addCssClasses(...)**.

Reply:
Yes, and this will highlight the duplication